src/gui/guiFormSpecMenu.cpp

```diff
@@ -26,6 +26,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "guiButton.h"
 #include "guiFormSpecMenu.h"
 #include "guiTable.h"
+#include "guiScrollBar.h"
```

Comment:
\*cough\* \*cough\* _…pqr**st**uvwxyz_

Reply:
They're supposed to be in alphabetical order?  I didn't know that.